lib/stat_tracker.rb

```diff
@@ -0,0 +1,84 @@
+require "csv"
+
+class StatTracker
+
+  def self.from_csv(locations)
+    StatTracker.new(locations)
+  end
+
+  attr_reader :games, :teams, :game_teams
+
+  def initialize(locations)
+    @games = create_games(locations[:games])
+    @teams = create_teams(locations[:teams])
+    @game_teams = create_game_teams(locations[:game_teams])
+  end
+
+# helper methods
+  def csv_reader(file_path)
```

Comment:
rename `csv_reader` to `read_csv` to keep method names as verbs, not nouns

```suggestion
  def read_csv(file_path)
```